### ISSUE TITLE
VideoView crashes when rendering frames before initialized 

### ISSUE THREAD

**Issue by aaalaniz** [twilio/video-quickstart-android#105]
A customer reported a crash when attempting to add a `VideoView` as a `VideoTrack` renderer before the `VideoView` has been added to the view hierarchy. 

**Log**

```
I: MediaCodecVideoDecoder: Frame stride and slice height: 512 x 384 
I: SurfaceViewRenderer: Reporting frame resolution changed to 480x360 with rotation 270 
W: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.os.Handler.post(java.lang.Runnable)' on a null object reference 
W: at org.webrtc.SurfaceViewRenderer.updateFrameDimensionsAndReportEvents(SurfaceViewRenderer.java:316) 
W: at org.webrtc.SurfaceViewRenderer.renderFrame(SurfaceViewRenderer.java:197) 
W: at com.twilio.video.VideoView.renderFrame(VideoView.java:127) 
W: at com.twilio.video.VideoTrack$VideoRendererCallbackAdapter.renderFrame(VideoTrack.java:123) 
E: # 
# Fatal error in ../../webrtc/sdk/android/src/jni/peerconnection_jni.cc, line 835 
# last system error: 0 
# Check failed: !jni()->ExceptionCheck() 
# 
# 
A: Fatal signal 6 (SIGABRT), code -6 in tid 12650 (IncomingVideoSt) 
W: releaseBuffer() track 0x55a8090cd0 disabled due to previous underrun, restarting
```

**Workaround**

Ensure that `VideoView` is added to view hierarchy before adding as a `VideoTrack` renderer.

**Comment by aaalaniz** [comment]
@dreamerns investigated this issue and I believe we fixed this bug in 1.0.0-beta16. Waiting on confirmation from developer who originally reported issue.

**Comment by abhinavtech** [comment]
I'm facing the very same issue, can someone please help?

**Comment by aaalaniz** [comment]
Hey @abhinavtech 

This issue is a little dated. Can you open a new issue and fill out the template?

Thanks!
